themes-default/slim/src/components/show-header.vue

```diff
@@ -21,24 +21,24 @@
                         >{{ show.title }}</app-link> / Season {{ season }}<template v-if="episode && manualSearchType !== 'season'"> Episode {{ episode }}</template>
                     </span>
                 </div>
-                <div v-if="type !== 'snatch-selection' && show.seasons && show.seasons.length >= 1" id="show-specials-and-seasons" class="pull-right">
-                    <span class="h2footer display-specials" v-if="show.seasons.find(season => ({ season }) => season === 0)">
+                <div v-if="type !== 'snatch-selection' && show.seasonCount && Object.keys(show.seasonCount).length > 1" id="show-specials-and-seasons" class="pull-right">
+                    <span class="h2footer display-specials" v-if="show.seasonCount['0']">
                         Display Specials: <a @click="toggleSpecials()" class="inner" style="cursor: pointer;">{{ displaySpecials ? 'Hide' : 'Show' }}</a>
                     </span>
 
                     <div class="h2footer display-seasons clear">
                         <span>
-                            <select v-if="show.seasons.length >= 15" v-model="jumpToSeason" id="seasonJump" class="form-control input-sm" style="position: relative">
+                            <select v-if="Object.keys(show.seasonCount) >= 15" v-model="jumpToSeason" id="seasonJump" class="form-control input-sm" style="position: relative">
                                 <option value="jump">Jump to Season</option>
-                                <option v-for="season in show.seasons" :key="'jumpToSeason-' + season[0].season" :value="'#season-' + season[0].season" :data-season="season[0].season">
-                                    {{ season[0].season === 0 ? 'Specials' : 'Season ' + season[0].season }}
+                                <option v-for="season in Object.keys(show.seasonCount)" :key="`jumpToSeason-${season}`" :value="`#season-${season}`" :data-season="season">
+                                    {{ season === 0 ? 'Specials' : 'Season ' + season }}
```

Comment:
`season` is a string, not a number.
```suggestion
                                    {{ season === '0' ? 'Specials' : 'Season ' + season }}
```